Jenkinsfile

```diff
@@ -58,74 +58,52 @@ node ('master') {
             '''
         }
 
+        // Set the ISOLATION_ID environment variable for the whole pipeline
+        env.ISOLATION_ID = sh(returnStdout: true, script: 'printf $BUILD_TAG | sha256sum | cut -c1-64').trim()
+
         // Use a docker container to build and protogen, so that the Jenkins
         // environment doesn't need all the dependencies.
         stage("Build Test Dependencies") {
-            docker.build('sawtooth-build:$BUILD_TAG', '-f docker/sawtooth-build-ubuntu-xenial .')
-            docker.withServer('tcp://0.0.0.0:4243'){
-                docker.image('sawtooth-build:$BUILD_TAG').inside {
-                    sh './bin/build_all'
-                }
-            }
+            sh './bin/build_all'
         }
 
         stage("Run Lint") {
-            docker.withServer('tcp://0.0.0.0:4243'){
-                docker.image('sawtooth-build:$BUILD_TAG').inside {
-                    sh './bin/run_lint'
-                }
-            }
+            sh 'docker run --rm -v $(pwd):/project/sawtooth-core sawtooth-build-python:$ISOLATION_ID ./bin/run_lint'
         }
 
         // Run the tests
         stage("Run Tests") {
-            sh 'ISOLATION_ID=$(printf $BUILD_TAG | sha256sum | cut -c1-64) ./bin/run_tests'
+            sh './bin/run_tests'
```

Comment:
Perhaps, until we get the java sdk properly working, we can have this skip the java sdk tests for now.